docs/data/getting-started/overview/overview.md

```diff
@@ -0,0 +1,37 @@
+---
+title: MUI X - Overview
+---
+
+# MUI X - Overview
+
+<p class="description">MUI X is a collection of advanced React UI components for complex use cases.</p>
+
+## Introduction
+
+MUI X is a collection of advanced UI components, including our [Data Grid](/x/react-data-grid) and [Date and Time Pickers](/x/react-date-pickers/getting-started).
```

Comment:
Ok, I also checked it and I have just one general note. 
On the [Overview page](https://deploy-preview-4692--material-ui-x.netlify.app/x/getting-started/overview/) right now there is no quick installation guide (just 2 lines explaining what package you need to install). To know this you need to from the Overview page to click on either component. This will take the user to [that components Overview page](https://deploy-preview-4692--material-ui-x.netlify.app/x/react-data-grid/) where you still can't see how to install the package. You then have to go to the [Getting Started page](https://deploy-preview-4692--material-ui-x.netlify.app/x/react-data-grid/getting-started/#installation) of that component and more specifically the installation section to find that information.
I think this is 2 steps too many. We can just add a quick installation guide to the general Overview page for MUI X
```
// with npm
npm install @mui/x-data-grid

// with yarn
yarn add @mui/x-data-grid
```